modules/ocf_puppet/manifests/puppetserver.pp

```diff
@@ -35,6 +35,17 @@
     notify               => Service['puppetserver'],
   }
 
+  puppet_authorization::rule { 'allow-puppetserver-cli':
```

Comment:
I'd add a comment above this that the point here is for the puppet master to be able to access puppet information about itself so that `puppetserver ca list` works (hence the `$::fqdn` in `allow`), since that wasn't clear to me when first reading this and I think it could confuse others too.